tests/CMakeLists.txt

```diff
@@ -117,6 +117,15 @@ if(AVIF_ENABLE_GTEST)
     target_include_directories(avify4mtest PRIVATE ${GTEST_INCLUDE_DIRS})
     add_test(NAME avify4mtest COMMAND avify4mtest)
 
+    if(NOT AVIF_CODEC_AOM OR NOT AVIF_CODEC_AOM_ENCODE OR NOT AVIF_CODEC_AOM_DECODE)
+        # These tests are supported with aom being the encoder and decoder. If aom is unavailable,
+        # these tests are disabled because other codecs may not implement all the necessary features.
+        # For example, SVT-AV1 requires 8-bit, 4:2:0 images with even dimensions of at least 64x64 px.
```

Comment:
Nit/optional: This comment could be misinterpreted to mean SVT-AV1 requires all of those conditions to be true. After reading this comment, I was wondering if SVT-AV1 did not support 10-bit. But I think it does.

Reply:
I got a segfault when encoding a 10-bit 64x64 4:2:0 image with SVT-AV1 but I did not take the time to investigate. Maybe I missed something. I removed the 8-bit part.

Reply:
SVT-AV1 supports 10-bit:
https://gitlab.com/AOMediaCodec/SVT-AV1/-/blob/master/Docs/CommonQuestions.md#8-or-10-bit-encoding

The segfault may mean there is a bug in libavif/src/codec_svt.c.